docs/configuration-reference.md

```diff
@@ -3729,3 +3731,99 @@ Full reference of options:
     # Maps to collector.logical_disk.volume-blacklist in windows_exporter
     [blacklist: <string> | default=".+"]
 ```
+
+### kafka_lag_exporter_config
+
+The `kafka_lag_exporter_config` block configures the `kafka_lag_exporter`
+integration, which is an embedded version of [`kafka_exporter`](https://github.com/davidmparrott/kafka_exporter).
+This allows for the collection of Kafka Lag metrics and exposing them as Prometheus metrics.
+
+Full reference of options:
+
+```yaml
+
+  #Address array (host:port) of Kafka server
+
+  [kafka_uris: [- string] | default="localhost:11091"]
+
+  #Connect using SASL/PLAIN
+  
+  [use_sasl: <bool> | default=false]
```

Comment:
You don't have to specify the default if it matches the Go zero values (false for bools, empty string for strings) :)